content/enterprise_influxdb/v1.9/administration/manage/users-and-permissions/authorization-api.md

```diff
@@ -504,8 +521,10 @@ curl --location-trusted --negotiate -u "admin:changeit" -s https://cluster_node_
 }
 ```
 
-##### Add permissions to a role
-Add permissions to a role.
+##### Add permissions to a role for all databases
+
+To grant a list of permissions to a role for all databases in a cluster,
```

Comment:
I feel like a comment like this should appear in the specific database case.  

Or maybe we need to break down what's in the JSON package for the API call?
use language to describe what you are trying to achieve and how this looks in the JSON structure.

In this example, we grant the "KapacitorAPI" and "KapacitorConfigAPI" permissions..... (btw... this is a really BAD example -- because these permissions are specific to Kapacitor and accessing Kapacitor).  We should use "Read", "Write", "DropData".    Or just "Write" and "DropData"....  

So in text... it would read something like:
In this example, we want to grant "Read", "Write", and "DropData" to the user named "phantom" for ALL databases within the cluster.